rpcd/playbooks/roles/rpc_maas/tasks/main.yml

```diff
@@ -60,6 +60,9 @@
 - include: swift.yml
   vars:
     external_vip_address: "{{ external_lb_vip_address }}"
+  when:
+    - defined groups['swift_proxy'][0]
+    - inventory_hostname == groups['swift_proxy'][0]
```

Comment:
This doesn't look right, but I'm not overly familiar w/ ansible.  I'd have expected this to be a simple `when: a and b`.  Can someone confirm?